exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
     handler: function () {
       return 'Good Evening to You!';
     },
+    //I want the bot to find the to be have a varity of moods when asked a question.
+
```

Comment:
Missing an opening curly brace here.
This could explain why your isListening is not working as it should on the next three prompts and why your second prompt isn't working either.